openshift/topics/get_started.adoc

```diff
@@ -20,14 +20,23 @@ To create a Registry Service Account, navigate to the link:https://access.redhat
 See the link:https://access.redhat.com/RegistryAuthentication[Red Hat Container Registry Authentication] article for more information.
 ====
 
-To update to the latest {project_openshift_product_name} images, run the following commands:
+. Ensure that you are logged in as a cluster administrator or a user with project administrator access to the global openshift project:
 
-. On your master host(s), ensure that you are logged in as a cluster administrator or a user with project administrator access to the global `openshift` project.
+. Choose a command based on your version of OpenShift Container Platform.
+
+.. If you are running an OpenShift Container Platform v3 based cluster instance on (some) of your master host(s), perform the following:
 +
 [source,bash,subs="attributes+,macros+"]
 ----
 $ oc login -u system:admin
 ----
+.. If you are running an OpenShift Container Platform v4 based cluster instance, link:https://docs.openshift.com/container-platform/latest/cli_reference/openshift_cli/getting-started-cli.html#cli-logging-in_cli-developer-commands[log in to the CLI] as the link:https://docs.openshift.com/container-platform/latest/authentication/remove-kubeadmin.html#understanding-kubeadmin_removing-kubeadmin[kubeadmin] user:
```

Comment:
@andymunro (Nitpick) would replace the `link:https://docs.openshift.com/container-platform/latest/cli_reference/openshift_cli/getting-started-cli.html#cli-logging-in_cli-developer-commands[log in to the CLI]` link with putting the URL into the [`document_attributes_product.adoc`](https://github.com/keycloak/keycloak-documentation/blob/master/topics/templates/document-attributes-product.adoc#L77) file maybe as `ocpdocs_logging_into_the_cli_link`.

Similarly for `link:https://docs.openshift.com/container-platform/latest/authentication/remove-kubeadmin.html#understanding-kubeadmin_removing-kubeadmin[kubeadmin]` link would place it into the same file (using same logic) probably as `ocpdocs_removing_kubeadmin_link`.

Reply:
Add a note before the following command yet:

_NOTE: In the following command be sure to replace the `password` argument with the actual password of the `kubeadmin` user, and the `openshift.example.com` argument with the actual domain name of the Kubernetes API server, specific to your cluster._